pydicom/values.py

```diff
@@ -150,18 +158,51 @@ def convert_DS_string(byte_string, is_little_endian, struct_format=None):
 
     Returns
     -------
-    valuerep.DSfloat or valuerep.DSdecimal or list of DSfloat/DSdecimal
-        If :attr:`~pydicom.config.use_DS_decimal` is ``True`` then returns
-        :class:`~pydicom.valuerep.DSdecimal` or a :class:`list` of
-        ``DSdecimal``, otherwise returns :class:`~pydicom.valuerep.DSfloat` or
-        a ``list`` of ``DSfloat``.
+    :class:`~pydicom.valuerep.DSfloat`, :class:`~pydicom.valuerep.DSdecimal`, :class:`numpy.float64`, list of DSfloat/DSdecimal or :class:`numpy.ndarray`   of :class:`numpy.float64`
+
+        If :attr:`~pydicom.config.use_DS_decimal` is ``False`` (default),
+        returns a :class:`~pydicom.valuerep.DSfloat` or list of them
+
+        If :attr:`~pydicom.config.use_DS_decimal` is ``True``,
+        returns a :class:`~pydicom.valuerep.DSdecimal` or list of them
+
+        If :data:`~pydicom.config.use_DS_numpy` is ``True``,
+        returns a :class:`numpy.float64` or a :class:`numpy.ndarray` of them
+
+    Raises
+    ------
+    ValueError
+        If :data:`~pydicom.config.use_DS_numpy` is ``True`` and the string
+        contains non-valid characters
+
+    ImportError
+        If :data:`~pydicom.config.use_DS_numpy` is ``True`` and numpy is not
+        available
+
+
+    .. versionchanged:: 2.0
```

Comment:
Nit-picking, but this should be at the bottom of the first part (i.e. before the Parameters section). Same with `convert_IS_string`